docs/applications/paperless_ng.md

```diff
@@ -0,0 +1,23 @@
+# Paperless-ng
+
+Homepage: [https://github.com/jonaswinkler/paperless-ng](https://github.com/jonaswinkler/paperless-ng)
+
+Paperless is an application by Daniel Quinn and contributors that indexes your scanned documents and allows you to easily search for documents and store metadata alongside your documents.
+
+Paperless-ng is a fork of the original project, adding a new interface and many other changes under the hood
+
+## Usage
+
+Set `paperless_enabled: true` in your `inventories/<your_inventory>/nas.yml` file.
+
+The paperless-ng web interface can be found at http://ansible_nas_host_or_ip:16922.
+
+### Create the superuser
+
+To be able to login, you will need a super user. To create it, execute the following command:
+
+```bash
+docker-compose run --rm ng_paperless_uiserver createsuperuser
```

Comment:
I'm not clear how this would work?

Reply:
It didn't, I just copy and pasted it from the documentation. Fixed now.